lib/helpers.js

```diff
@@ -7,11 +7,13 @@ const EconItem = require('./classes/EconItem.js');
 const TradeOffer = require('./classes/TradeOffer.js');
 const EConfirmationMethod = require('../resources/EConfirmationMethod.js');
 
-exports.itemEquals = function(a, b) {
-	return a.appid == b.appid && a.contextid == b.contextid && (a.assetid || a.id) == (b.assetid || b.id);
+exports.itemEquals = function (a, b) {
+	return a.appid === b.appid
```

Comment:
I foresee this breaking stuff because we could very well end up with a string vs int matching going on here.